src/document.h

```diff
@@ -1313,6 +1320,17 @@ struct Document {
                 }
                 return nullptr;
 
+            case A_PROGRESSCELL: {

```

Comment:
Looks like this case could be merged with the case above it, with only `A_DOWN` and `A_RIGHT` being different?

Reply:
Agreed - both handlers are merged now.